server-apollo/src/schema.js

```diff
@@ -49,18 +49,22 @@ const resolvers = {
       return result
     },
     updateUser: async (obj, args, context, info) => {
-      let { id, version, ...updateArgs } = args
-      const currentRecord = await context.db('users').select().where('id', args.id).then((rows) => rows[0])
-      if (!currentRecord) return null // or not found error??
+      return new Promise((resolve, reject) => {
```

Comment:
Let's move this to SDK as separate resolve and merge it so we will have ready to go example for our own server

Reply:
+1 good idea